playbooks/build-ansible-collection/run.yaml

```diff
@@ -31,6 +31,10 @@
         patterns: "*.tar.gz"
       register: result
 
+    - name: Install the Collections locally, to be sure the dependencies get pulled
+      command: "{{ ansible_galaxy_executable }} collection install --force {{ item.path }}"
+      loop: "{{ result.files }}"
+
     - name: Confirm collection can be imported into galaxy
```

Comment:
just to follow up here, I think this needs to move into its own job. otherwise, this current job will get way more complicated. And given it is trusted, makes is much harder to test.

We can leverage the existing download-collections role, to first fetch and install the required collections, before calling galaxy-importer.